provisioning/transport/http/src/ApiVersionDelegatingHandler.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace Microsoft.Azure.Devices.Provisioning.Client.Transport.Http
+{
+    internal class ApiVersionDelegatingHandler : DelegatingHandler
+    {
+        private const string ApiVersion = "2017-08-31-preview";
```

Comment:
api version should be common to protocols.  We have a common file for that....TODO for cleanup